color-coordinated/color-picker-fe
# Delete project

#### What's this PR do?
Adds the ability to delete a palette and a project. Converts save palette form to use state instead of event.target
#### Where should the reviewer start?
#### How should this be manually tested?
Once BE is updated when you click delete icon for palette it should be deleted and persist. When a project is deleted a dialog should confirm delete. Project and palettes associated with it should be removed and persist.
#### Any background context you want to provide?
Currently delete does not persist because it is set up to use ID and not name. So once we push latest changes on BE to heroku deleting projects and palettes should persist. Didn't do any styling yet will get to that tomorrow.
